contracts/Pod.sol

```diff
@@ -47,76 +47,66 @@ contract Pod is
     |__________________________________*/
     IERC20Upgradeable public token;
     IERC20Upgradeable public ticket;
-    IERC20Upgradeable public pool;
+    IERC20Upgradeable public reward;
 
     // Initialized Contracts
     TokenFaucet public faucet;
     TokenDrop public drop;
 
-    // Private
-    IPrizePool private _prizePool;
-
     // Manager
     IPodManager public manager;
 
+    // Private
+    IPrizePool private _prizePool;
```

Comment:
Since anything can be the manager, let's make this type simply address

Reply:
Ahhh yes - thank for the reminder! I actually wanted to ask you about the `supportsInterface()` and if you that might be needed for anything and should be updated to actually do something.

Reply:
> Since anything can be the manager, let's make this type simply address

Duh... Don't know why I use an interface for that.